jenkins/deploy/scripts/install_compilers_parallel.sh

```diff
@@ -46,6 +46,17 @@ do
     fi
 done <<< $(cat compilers.list)
 
+# Hack for intel compiler.
+# We want to add the following line to ifort.cfg (overwrite).
+# -isystem/ssoft/spack/syrah/v1/opt/spack/linux-rhel8-x86_64_v2/gcc-8.5.0/intel-oneapi-compilers-classic-2021.6.0-q3mi2mylw3zyuht6p72u25ruqnpptpym/compiler/include/intel64
+DST_PRFX=`spack location -i intel-oneapi-compilers`
```

Comment:
why not using `spack location -i intel-oneapi-compilers-classic`/bin/ifort.cfg
has the destination ?

Reply:
You're right, I've got the wrong path. ```DST_PRFX=`spack location -i intel-oneapi-compilers`/bin``` is fine.

Reply:
I have added a new commit following your comment.